### ISSUE TITLE
When using CompanyCurrency it throws exception: AttributeError: 'MetaData' object has no attribute 'from_json'

### ISSUE THREAD

**Issue by antetna** [ej2/python-quickbooks#156]
What am I doing wrong with this because other objects are working just fine but not this one CompanyCurrency

```
from quickbooks import QuickBooks
client = QuickBooks(
         auth_client=auth_client,
         refresh_token=REFRESH_TOKEN,
         company_id=REALM_ID,
     )
from quickbooks.objects.companycurrency import CompanyCurrency
cc = CompanyCurrency().all(qb=client)
```

```
Traceback (most recent call last):
  File "/home/user/myprojects/quickbooks/main.py", line 35, in <module>
    cc = CompanyCurrency().all(qb=client)
  File "/home/user/myprojects/quickbooks/venv/lib/python3.6/site-packages/quickbooks/mixins.py", line 184, in all
    max_results=max_results, qb=qb)
  File "/home/user/myprojects/quickbooks/venv/lib/python3.6/site-packages/quickbooks/mixins.py", line 235, in where
    return cls.query(select, qb=qb)
  File "/home/user/myprojects/quickbooks/venv/lib/python3.6/site-packages/quickbooks/mixins.py", line 253, in query
    obj_list.append(cls.from_json(item_json))
  File "/home/user/myprojects/quickbooks/venv/lib/python3.6/site-packages/quickbooks/mixins.py", line 31, in from_json
    sub_obj = sub_obj.from_json(json_data[key])
AttributeError: 'MetaData' object has no attribute 'from_json'
```

_Originally posted by @antetna in https://github.com/sidecars/python-quickbooks/issues/135#issuecomment-508808781_

**Comment by ej2** [comment]
You aren't doing anything wrong... there is an issue with the code. I am working on a fix and writing tests but it might be a few days before I release it. The main problem is the MetaData object on CompanyCurrency needs to be removed.

**Comment by antetna** [comment]
I removed MetaData and CustomField from CompanyCurrency and I was able to retireve data, but than there was another problem when trying to create customer with currency, it always defaults to main company  currency, it is like nothing is sent  to API regarding currency info:

```
cur = CompanyCurrency.filter(Active=True, Code='GBP', qb=client)
# cur[0].to_ref() return "GBP"
 
c = Customer()
c.GivenName = 'John'
c.FamilyName = 'Doe'
c.CurrencyRef = cur[0].to_ref() 
c.save(qb=client)
```
Currency is not assigned to customer after checking it in quickbooks, it defaults to company main currency.

**Comment by antetna** [comment]
any progress ?

**Comment by swapnilsurdi** [comment]
any progress?

**Comment by ej2** [comment]
I am working on an update for the original issue with the Metadata. The CurrencyRef is a **readonly** field and cannot be updated according to the intuit documentation: https://developer.intuit.com/app/developer/qbo/docs/api/accounting/all-entities/customer

**Comment by ej2** [comment]
Fixed in 0.8.3 - https://github.com/ej2/python-quickbooks/releases/tag/0.8.3

**Comment by swapnilsurdi** [comment]
Thank you
